lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -531,22 +556,29 @@ TypeLoweringVisitor::addArg(Operation *module, unsigned insertPt,
     newValue = body->insertArgument(insertPt, field.type);
   }
 
+  auto *ctxt = builder->getContext();
   // Save the name attribute for the new argument.
   auto name =
       builder->getStringAttr(oldArg.name.getValue().str() + field.suffix.str());
 
+  StringAttr sym;
+  if (oldArg.sym)
+    sym = StringAttr::get(ctxt, oldArg.sym.getValue() + field.suffix.str());
+  else
+    sym = StringAttr::get(ctxt,
+                          (oldArg.name.getValue().str() + field.suffix.str()));
```

Comment:
This will probably need to be uniqued in the namespace to avoid collisions, yes?

Reply:
Right, I'm planning to do this in a subsequent PR.